test/sasdataloader/test/utest_sesans.py

```diff
@@ -0,0 +1,100 @@
+"""
+    Unit tests for the SESANS .ses reader
+"""
+
+import unittest
+from sas.sascalc.dataloader.readers.sesans_reader import Reader
+
+
+class sesans_reader(unittest.TestCase):
+
+    def setUp(self):
+        reader = Reader()
+        self.loader = reader.read
+
```

Comment:
Can you also use Loader.load and check that the default loader finds the correct file?